apps/builder/app/builder/features/sidebar-left/panels/pages/url-pattern.ts

```diff
@@ -113,3 +113,7 @@ export const validatePathnamePattern = (pathname: string) => {
 
   return messages;
 };
+
+export const validateSlug = (slug: string) => {
+  return [];
+};
```

Comment:
what is does?

Reply:
nothing yet, will add. a todo

Reply:
actually don't need that at all since schema is already validating all we need